Constellation/Node/Api.hs

```diff
@@ -113,6 +139,38 @@ data Whitelist = Whitelist
     , wlIPv6 :: Set HostAddress6
     } deriving Show
 
+decodeSendRaw :: BL.ByteString -> RequestHeaders -> Either String Send
+decodeSendRaw b h = case getHeaders [hContentLength, hFrom, hTo] h of
+    Right headers -> Right Send
+                    { sreqPayload = decodePayload (hmap ! hContentLength) b
+                    , sreqFrom    = mustDecodeB64PublicKey $ hmap ! hFrom
```

Comment:
I'd prefer to use the headers from http-types as that's what they're there for (even if only in this file). However, happy to go with constants if you feel strongly on this.